databuilder/publisher/elasticsearch_publisher.py

```diff
@@ -87,6 +87,9 @@ class ElasticsearchPublisher(Publisher):
                 "tags": {
                   "type": "keyword"
                 },
+                "badges": {
+                  "type": "keyword"
+                },
                 "cluster": {
```

Comment:
Was not clear to me if I should edit this. I think it should allow users to filter by badge via ES syntax